website/src/components/DemoPage/CodeStream/CodeStream.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+
+import SplitDemo from '@site/src/components/DemoPage/SplitDemo'
+import DemoTabs from '@site/src/components/DemoPage/DemoTabs'
+import TextBlock from '@site/src/components/DemoPage/TextBlock'
+
+import Local from '@site/src/components/DemoPage/Overview/Local'
+import Deploy from '@site/src/components/DemoPage/Overview/Deploy'
+import Resources from '@site/src/components/DemoPage/Overview/Resources'
+
+import { DataDemos } from '../../../DataDemos'
+
+const DataDemo = DataDemos.find(item => item.detailURL === 'demos/code-streaming')
+
+const languages = DataDemo.languages
+const githubURL = DataDemo.githubRepo
+
+function CodeStream() {
+  return (
+    <>
+      <SplitDemo
+        leftSrc="https://awps-demos-codestream.azurewebsites.net/"
+        rightSrc="https://awps-demos-codestream.azurewebsites.net/"
+        description="Real-time chat app demo"
+        width="400"
+        languages={languages}
+        githubURL={githubURL}
+      />
+
+      <div className="max-w-full overflow-hidden">
+        <DemoTabs overview={<Overview />} local={<Local />} resources={<Resources />} />
+      </div>
+    </>
+  )
+}
+
+function Overview() {
+  return (
+    <div>
+      <h2 className="text-4xl">Overview</h2>
```

Comment:
dosen't auto convert from demo readme, please make it auto in the future

Reply:
Will do. I will implement the plugin.